src/types/InvokerOptions.type.ts

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Please also add this to `src/index.ts`, otherwise users won't be able to use this.